rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/notifications/OpenUnityProjectAsFolderNotification.kt

```diff
@@ -31,14 +34,19 @@ class OpenUnityProjectAsFolderNotification(private val project: Project, private
 
         val solutionDescription = project.solutionDescription
         if (solutionDescription is RdVirtualSolution) {
-
+var reopenText = "<a href=\"reopen\">reopen</a>"
+            val (status, editorInstanceJson) = EditorInstanceJson.load(project)
+            if (status == EditorInstanceJsonStatus.Valid) {
+                 // do not show Action
+                reopenText = "reopen"
+            }
             val content = if (solutionDescription.projectFilePaths.isEmpty()) {
                 "This looks like a Unity project. C# and Unity specific functionality is not available when the project is opened as a folder." +
-                    " Please <a href=\"close\">close</a> and reopen through the Unity editor, or by opening a .sln file."
+                    " Please <a href=\"close\">close</a> and "+reopenText+" through the Unity editor, or by opening a .sln file."
```

Comment:
We can only click on one link before the notification is gone, so it's going to be hard for the user to choose between "close" and "reopen".

So if we have `EditorInstance.json` (Unity is currently running), leave it as it was:

```
Please <a href=\"close\">close</a> and reopen through the Unity editor, or by opening a .sln file.
```

But if we don't have it (Unity isn't running), we should probably only have one action link:

```
Please <a href=\"reopen\">click here</a> to start Unity, generate a solution file and reopen the project.
```